semcore/wizard/__tests__/index.test.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { testing, snapshot } from '@semcore/jest-preset-ui';
+import Wizard from '../src';
+
+const { cleanup, render, axe } = testing;
+
+const steps = [
+  { value: 1, title: 'Step 1', disabled: false },
+  { value: 2, title: 'Step 2', disabled: false },
+];
+
+describe('Wizard', () => {
+  afterEach(cleanup);
+
+  test('Should correctly render', async () => {
+    const Component = (
+      <Wizard currentStep={1} steps={steps}>
```

Comment:
maybe replace `currentStep` to `value` and `defaultValue` what this did in other components?

Reply:
State here is very different from "value" state in other example. I am voting for current naming